spec/helpers/authentication_helper_spec.rb

```diff
@@ -38,33 +38,6 @@
     end
   end
 
-  describe "#recaptcha_configured_and_enabled?" do
```

Comment:
This test was migrated to `spec/services/re_captcha_spec.rb` (not entirely removed). So the coverage regarding this check remains the same.